Description: cleaning up upstream tarball complaints
 We should probably remove explicit paths from Makefiles completely
 .
 xf86-input-cmt (2.0.25) xenon; urgency=medium
 .
   * fix postinst failure on build server
Author: reynhout <reynhout@galliumos.org>

---
--- xf86-input-cmt-2.0.25.orig/man/Makefile
+++ xf86-input-cmt-2.0.25/man/Makefile
@@ -148,7 +148,7 @@ am__can_run_installinfo = \
   esac
 am__tagged_files = $(HEADERS) $(SOURCES) $(TAGS_FILES) $(LISP)
 DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
-ACLOCAL = ${SHELL} /home/hugh/Downloads/xf86-input-cmt/missing aclocal-1.14
+ACLOCAL = ${SHELL} /home/hugh/Downloads/to_checkin/xf86-input-cmt/missing aclocal-1.14
 ADMIN_MAN_DIR = $(mandir)/man$(ADMIN_MAN_SUFFIX)
 ADMIN_MAN_SUFFIX = 8
 AMTAR = $${TAR-tar}
@@ -156,9 +156,9 @@ AM_DEFAULT_VERBOSITY = 1
 APP_MAN_DIR = $(mandir)/man$(APP_MAN_SUFFIX)
 APP_MAN_SUFFIX = 1
 AR = ar
-AUTOCONF = ${SHELL} /home/hugh/Downloads/xf86-input-cmt/missing autoconf
-AUTOHEADER = ${SHELL} /home/hugh/Downloads/xf86-input-cmt/missing autoheader
-AUTOMAKE = ${SHELL} /home/hugh/Downloads/xf86-input-cmt/missing automake-1.14
+AUTOCONF = ${SHELL} /home/hugh/Downloads/to_checkin/xf86-input-cmt/missing autoconf
+AUTOHEADER = ${SHELL} /home/hugh/Downloads/to_checkin/xf86-input-cmt/missing autoheader
+AUTOMAKE = ${SHELL} /home/hugh/Downloads/to_checkin/xf86-input-cmt/missing automake-1.14
 AWK = gawk
 BASE_CFLAGS =  -Wall -Wpointer-arith -Wmissing-declarations -Wformat=2 -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Wbad-function-cast -Wold-style-definition -Wdeclaration-after-statement -Wunused -Wuninitialized -Wshadow -Wcast-qual -Wmissing-noreturn -Wmissing-format-attribute -Wredundant-decls -Werror=implicit -Werror=nonnull -Werror=init-self -Werror=main -Werror=missing-braces -Werror=sequence-point -Werror=return-type -Werror=trigraphs -Werror=array-bounds -Werror=write-strings -Werror=address -Werror=int-to-pointer-cast -Werror=pointer-to-int-cast
 CC = gcc
@@ -203,7 +203,7 @@ LIPO =
 LN_S = ln -s
 LTLIBOBJS = 
 MAINT = #
-MAKEINFO = ${SHELL} /home/hugh/Downloads/xf86-input-cmt/missing makeinfo
+MAKEINFO = ${SHELL} /home/hugh/Downloads/to_checkin/xf86-input-cmt/missing makeinfo
 MANIFEST_TOOL = :
 MAN_SUBSTS = 	-e 's|__vendorversion__|"$(PACKAGE_STRING)" "$(XORG_MAN_PAGE)"|' 	-e 's|__xorgversion__|"$(PACKAGE_STRING)" "$(XORG_MAN_PAGE)"|' 	-e 's|__xservername__|Xorg|g' 	-e 's|__xconfigfile__|xorg.conf|g' 	-e 's|__projectroot__|$(prefix)|g' 	-e 's|__apploaddir__|$(appdefaultdir)|g' 	-e 's|__appmansuffix__|$(APP_MAN_SUFFIX)|g' 	-e 's|__drivermansuffix__|$(DRIVER_MAN_SUFFIX)|g' 	-e 's|__adminmansuffix__|$(ADMIN_MAN_SUFFIX)|g' 	-e 's|__libmansuffix__|$(LIB_MAN_SUFFIX)|g' 	-e 's|__miscmansuffix__|$(MISC_MAN_SUFFIX)|g' 	-e 's|__filemansuffix__|$(FILE_MAN_SUFFIX)|g'
 MISC_MAN_DIR = $(mandir)/man$(MISC_MAN_SUFFIX)
@@ -236,10 +236,10 @@ VERSION = 0.0.1
 XORG_CFLAGS = -fvisibility=hidden -I/usr/include/pixman-1 -I/usr/include/libdrm -I/usr/include/xorg -I/usr/include/X11/dri
 XORG_LIBS = 
 XORG_MAN_PAGE = X Version 11
-abs_builddir = /home/hugh/Downloads/xf86-input-cmt/man
-abs_srcdir = /home/hugh/Downloads/xf86-input-cmt/man
-abs_top_builddir = /home/hugh/Downloads/xf86-input-cmt
-abs_top_srcdir = /home/hugh/Downloads/xf86-input-cmt
+abs_builddir = /home/hugh/Downloads/to_checkin/xf86-input-cmt/man
+abs_srcdir = /home/hugh/Downloads/to_checkin/xf86-input-cmt/man
+abs_top_builddir = /home/hugh/Downloads/to_checkin/xf86-input-cmt
+abs_top_srcdir = /home/hugh/Downloads/to_checkin/xf86-input-cmt
 ac_ct_AR = ar
 ac_ct_CC = gcc
 ac_ct_DUMPBIN = 
@@ -269,7 +269,7 @@ htmldir = ${docdir}
 includedir = ${prefix}/include
 infodir = ${prefix}/share/info
 inputdir = ${exec_prefix}/lib/xorg/modules/input
-install_sh = ${SHELL} /home/hugh/Downloads/xf86-input-cmt/install-sh
+install_sh = ${SHELL} /home/hugh/Downloads/to_checkin/xf86-input-cmt/install-sh
 libdir = ${exec_prefix}/lib
 libexecdir = ${prefix}/lib/xf86-input-cmt
 localedir = ${datarootdir}/locale
